src/WebJobs.Script/Diagnostics/FunctionTraceWriterFactory.cs

```diff
@@ -22,12 +24,17 @@ public TraceWriter Create()
         {
```

Comment:
Sorry, I should have added a comment in the PR. It's an improvement when using ILogger. The function-level logs that come from the invoker like "Function started" are logged by a TraceWriter, but from within the function, you can log with an ILogger. Most of the time, these would go to the same file -- as long as there wasn't a race at startup -- but during testing I would sometimes see two files created because they were both checking for an existing file, not finding one, so creating one themselves.

This removes all of that by making both the ILogger and the TraceWriter paths use the same underlying FileTraceWriter.